src/Bicep.LangServer/Bicep.LangServer.csproj

```diff
@@ -13,7 +13,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.AI.OpenAI" Version="1.0.0-beta.7" />
     <PackageReference Include="Azure.ResourceManager.ResourceGraph" Version="1.0.1" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.5.0" />
+    <PackageReference Include="Microsoft.WindowsAzure.Governance.PolicyService.Common" Version="1.10.2442.158" />
+    <PackageReference Include="Microsoft.WindowsAzure.Governance.PolicyService.Policy.Common" Version="1.10.2442.158" />
+    <PackageReference Include="Microsoft.WindowsAzure.Governance.PolicyService.Policy.DataClient" Version="1.10.2442.158" />
+    <PackageReference Include="Microsoft.WindowsAzure.Governance.PolicyService.Policy.Evaluation" Version="1.10.2442.158" />
+    <PackageReference Include="Microsoft.WindowsAzure.Governance.PolicyService.Policy.Expression" Version="1.10.2442.158" />
```

Comment:
none of these packages seem relevant to what we are looking to do so any references to them and any code that exercises them should be removed.